pkg/downloader/manager_test.go

```diff
@@ -657,7 +658,17 @@ func TestDedupeRepos(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := dedupeRepos(tt.repos); !reflect.DeepEqual(got, tt.want) {
+			got := dedupeRepos(tt.repos)
+
+			// Sort to compare ignoring order
+			sort.Slice(got, func(i, j int) bool {
+				return got[i].URL < got[j].URL
+			})
+			sort.Slice(tt.want, func(i, j int) bool {
+				return tt.want[i].URL < tt.want[j].URL
+			})
+
+			if !reflect.DeepEqual(got, tt.want) {
 				t.Errorf("received:\n%v\nwant:\n%v", got, tt.want)
 			}
```

Comment:
try: [assert.ElementsMatch](https://pkg.go.dev/github.com/stretchr/testify/assert#ElementsMatch)

```suggestion
			assert.ElementsMatch(t, tt.want, got)
```

Reply:
That works, but I had to add `github.com/tj/assert` to the list of dependencies. If that's undesirable, I can revert.

Reply:
ah, I was just writing https://github.com/helm/helm/pull/30576#pullrequestreview-2635723822

 it should be: `"github.com/stretchr/testify/assert"`